### DISCUSSION TITLE
vaultwarden iptable rules help ! limit ip:port access

### DISCUSSION THREAD

**Discussion by RatedR9** [dani-garcia/vaultwarden#2452] [Q&A] [answered]
I set iptables only local host can access but smtp 587 can’t connect,so how to set iptables rules correctly
here is my iptables rules:
iptables -A INPUT -p tcp -s localhost --dport 8000 -j ACCEPT (8000 is map port)
iptables -A INPUT -p tcp --dport 8000 -j DROP
Thanks for help.

**Comment by RealOrangeOne** [comment, marked as answer]
If you just want access to vaultwarden on localhost, why not just change what interface it binds to and leave it at that, no firewall config needed. `ROCKET_ADDRESS=127.0.0.1` and `WEBSOCKET_ADDRESS=127.0.0.1` should do the trick
